update create parameter topic (#269)
DevExpress/dotnet-eud: eud-for-devexpress-reports/reporting-for-web/report-designer/use-report-parameters/date-range-report-parameters.md

This topic describes how to create date and time range parameters and filter a report's data by the specified dates.

|Date Range Parameter Editor| Time Range Parameter Editor|
|--|--|
|![A date range parameter's editor in Preview](../../images/web-report-date-range-params.png)|![A time range parameter's editor in Preview](../../images/web-report-time-range-param.png)|

## Create a Range Parameter in the Report Designer

Follow the steps below to add a range parameter to a report in the [Report Designer](../first-look-at-the-report-designer.md):

1. [Create a report parameter](create-a-report-parameter.md). In the **Add Parameter** dialog, specify parameter options:

    - **Parameter type**: _Date and Time_,  _Date_, or _Time_
    - **Value Source**: _Range Parameters_
    

>     The **Start Parameter** and **End Parameter** sections that appear allow you to configure options to create a date or time range:

Reviewer: ```suggestion
    The **Start Parameter** and **End Parameter** sections allow you to configure options for a date or time range:
```
